cmd/omegaup-grader/frontend_handler.go

```diff
@@ -320,6 +327,36 @@ func broadcastRun(
 	if err != nil {
 		return err
 	}
+	var groupName string
+	var scoreByGroup float64
+	rows, error := queryWithRetry(
+		db,
+		`SELECT
+			rg.group_name, rg.score
+		FROM
+			Runs_Groups rg
+		WHERE
+			rg.run_id = ?;`, run.ID)
+	if error != nil {
+		return error
```

Comment:
```suggestion
		return fmt.Errorf("failed to query run groups: %w", err)
```

siempre hay que intentar dar contexto de por qué fallan las cosas.

Reply:
Listo.